Doc/library/asyncio-task.rst

```diff
@@ -520,7 +520,10 @@ Shielding From Cancellation
 
    The statement::
 
-       res = await shield(something())
+       task = asyncio.create_task(something())
+       background_tasks.add(task)
+       task.add_done_callback(background_tasks.discard)
```

Comment:
I feel that it's not our job here to educate users on how to manage collections of tasks that are automatically updated when the task completes. So let's remove all mention of `background_tasks` here and below.
```suggestion
```